2024-05-28 11:51:04,058 INFO    StreamThr :46677 [internal.py:wandb_internal():88] W&B internal server running at pid: 46677, started at: 2024-05-28 11:51:04.056472
2024-05-28 11:51:04,063 DEBUG   HandlerThread:46677 [handler.py:handle_request():158] handle_request: status
2024-05-28 11:51:04,067 INFO    WriterThread:46677 [datastore.py:open_for_write():87] open: /home/alessiatortone/ProjectMLDL/wandb/run-20240528_115104-kybjttf5/run-kybjttf5.wandb
2024-05-28 11:51:04,067 DEBUG   SenderThread:46677 [sender.py:send():378] send: header
2024-05-28 11:51:04,076 DEBUG   SenderThread:46677 [sender.py:send():378] send: run
2024-05-28 11:51:04,609 INFO    SenderThread:46677 [dir_watcher.py:__init__():211] watching files in: /home/alessiatortone/ProjectMLDL/wandb/run-20240528_115104-kybjttf5/files
2024-05-28 11:51:04,610 INFO    SenderThread:46677 [sender.py:_start_run_threads():1126] run started: kybjttf5 with start time 1716889864.055693
2024-05-28 11:51:04,622 DEBUG   HandlerThread:46677 [handler.py:handle_request():158] handle_request: check_version
2024-05-28 11:51:04,623 DEBUG   SenderThread:46677 [sender.py:send_request():405] send_request: check_version
2024-05-28 11:51:04,729 DEBUG   HandlerThread:46677 [handler.py:handle_request():158] handle_request: run_start
2024-05-28 11:51:04,730 DEBUG   HandlerThread:46677 [system_info.py:__init__():26] System info init
2024-05-28 11:51:04,730 DEBUG   HandlerThread:46677 [system_info.py:__init__():41] System info init done
2024-05-28 11:51:04,730 INFO    HandlerThread:46677 [system_monitor.py:start():194] Starting system monitor
2024-05-28 11:51:04,730 INFO    SystemMonitor:46677 [system_monitor.py:_start():158] Starting system asset monitoring threads
2024-05-28 11:51:04,731 INFO    HandlerThread:46677 [system_monitor.py:probe():214] Collecting system info
2024-05-28 11:51:04,731 INFO    SystemMonitor:46677 [interfaces.py:start():188] Started cpu monitoring
2024-05-28 11:51:04,732 INFO    SystemMonitor:46677 [interfaces.py:start():188] Started disk monitoring
2024-05-28 11:51:04,732 INFO    SystemMonitor:46677 [interfaces.py:start():188] Started memory monitoring
2024-05-28 11:51:04,733 INFO    SystemMonitor:46677 [interfaces.py:start():188] Started network monitoring
2024-05-28 11:51:04,742 DEBUG   HandlerThread:46677 [system_info.py:probe():150] Probing system
2024-05-28 11:51:04,766 DEBUG   HandlerThread:46677 [system_info.py:_probe_git():135] Probing git
2024-05-28 11:51:04,778 DEBUG   HandlerThread:46677 [system_info.py:_probe_git():143] Probing git done
2024-05-28 11:51:04,778 DEBUG   HandlerThread:46677 [system_info.py:probe():198] Probing system done
2024-05-28 11:51:04,778 DEBUG   HandlerThread:46677 [system_monitor.py:probe():223] {'os': 'Linux-5.15.146.1-microsoft-standard-WSL2-x86_64-with-debian-bookworm-sid', 'python': '3.7.12', 'heartbeatAt': '2024-05-28T09:51:04.742759', 'startedAt': '2024-05-28T09:51:04.037274', 'docker': None, 'cuda': None, 'args': (), 'state': 'running', 'program': 'test_sb3.py', 'codePathLocal': 'test_sb3.py', 'codePath': 'test_sb3.py', 'git': {'remote': 'https://github.com/lottiful/Project.git', 'commit': '1bc8dbc791145508e6b1cc1512634634e6d512cd'}, 'email': 'alessia.tortone@gmail.com', 'root': '/home/alessiatortone/ProjectMLDL', 'host': 'DESKTOP-C7KS0F6', 'username': 'alessiatortone', 'executable': '/home/alessiatortone/miniconda3/envs/mldl/bin/python', 'cpu_count': 4, 'cpu_count_logical': 8, 'cpu_freq': {'current': 2112.007, 'min': 0.0, 'max': 0.0}, 'cpu_freq_per_core': [{'current': 2112.007, 'min': 0.0, 'max': 0.0}, {'current': 2112.007, 'min': 0.0, 'max': 0.0}, {'current': 2112.007, 'min': 0.0, 'max': 0.0}, {'current': 2112.007, 'min': 0.0, 'max': 0.0}, {'current': 2112.007, 'min': 0.0, 'max': 0.0}, {'current': 2112.007, 'min': 0.0, 'max': 0.0}, {'current': 2112.007, 'min': 0.0, 'max': 0.0}, {'current': 2112.007, 'min': 0.0, 'max': 0.0}], 'disk': {'/': {'total': 1006.853931427002, 'used': 10.476543426513672}}, 'memory': {'total': 3.720775604248047}}
2024-05-28 11:51:04,778 INFO    HandlerThread:46677 [system_monitor.py:probe():224] Finished collecting system info
2024-05-28 11:51:04,778 INFO    HandlerThread:46677 [system_monitor.py:probe():227] Publishing system info
2024-05-28 11:51:04,778 DEBUG   HandlerThread:46677 [system_info.py:_save_conda():208] Saving list of conda packages installed into the current environment
2024-05-28 11:51:05,610 INFO    Thread-12 :46677 [dir_watcher.py:_on_file_created():271] file/dir created: /home/alessiatortone/ProjectMLDL/wandb/run-20240528_115104-kybjttf5/files/conda-environment.yaml
2024-05-28 11:51:06,509 DEBUG   HandlerThread:46677 [system_info.py:_save_conda():222] Saving conda packages done
2024-05-28 11:51:06,510 INFO    HandlerThread:46677 [system_monitor.py:probe():229] Finished publishing system info
2024-05-28 11:51:06,518 DEBUG   SenderThread:46677 [sender.py:send():378] send: files
2024-05-28 11:51:06,518 INFO    SenderThread:46677 [sender.py:_save_file():1389] saving file wandb-metadata.json with policy now
2024-05-28 11:51:06,598 DEBUG   HandlerThread:46677 [handler.py:handle_request():158] handle_request: python_packages
2024-05-28 11:51:06,599 DEBUG   SenderThread:46677 [sender.py:send_request():405] send_request: python_packages
2024-05-28 11:51:06,599 DEBUG   HandlerThread:46677 [handler.py:handle_request():158] handle_request: stop_status
2024-05-28 11:51:06,600 DEBUG   SenderThread:46677 [sender.py:send_request():405] send_request: stop_status
2024-05-28 11:51:06,609 INFO    Thread-12 :46677 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/alessiatortone/ProjectMLDL/wandb/run-20240528_115104-kybjttf5/files/conda-environment.yaml
2024-05-28 11:51:06,609 INFO    Thread-12 :46677 [dir_watcher.py:_on_file_created():271] file/dir created: /home/alessiatortone/ProjectMLDL/wandb/run-20240528_115104-kybjttf5/files/wandb-metadata.json
2024-05-28 11:51:06,610 INFO    Thread-12 :46677 [dir_watcher.py:_on_file_created():271] file/dir created: /home/alessiatortone/ProjectMLDL/wandb/run-20240528_115104-kybjttf5/files/requirements.txt
2024-05-28 11:51:07,695 DEBUG   HandlerThread:46677 [handler.py:handle_request():158] handle_request: partial_history
2024-05-28 11:51:08,090 DEBUG   HandlerThread:46677 [handler.py:handle_request():158] handle_request: partial_history
2024-05-28 11:51:08,577 DEBUG   HandlerThread:46677 [handler.py:handle_request():158] handle_request: partial_history
2024-05-28 11:51:08,867 DEBUG   HandlerThread:46677 [handler.py:handle_request():158] handle_request: partial_history
2024-05-28 11:51:09,226 DEBUG   HandlerThread:46677 [handler.py:handle_request():158] handle_request: partial_history
2024-05-28 11:51:09,725 DEBUG   HandlerThread:46677 [handler.py:handle_request():158] handle_request: partial_history
2024-05-28 11:51:10,084 DEBUG   HandlerThread:46677 [handler.py:handle_request():158] handle_request: partial_history
2024-05-28 11:51:10,445 DEBUG   HandlerThread:46677 [handler.py:handle_request():158] handle_request: partial_history
2024-05-28 11:51:10,966 DEBUG   HandlerThread:46677 [handler.py:handle_request():158] handle_request: partial_history
2024-05-28 11:51:11,360 DEBUG   HandlerThread:46677 [handler.py:handle_request():158] handle_request: partial_history
2024-05-28 11:51:11,643 DEBUG   HandlerThread:46677 [handler.py:handle_request():158] handle_request: keepalive
2024-05-28 11:51:11,803 DEBUG   HandlerThread:46677 [handler.py:handle_request():158] handle_request: partial_history
2024-05-28 11:51:12,263 DEBUG   HandlerThread:46677 [handler.py:handle_request():158] handle_request: partial_history
2024-05-28 11:51:12,646 DEBUG   HandlerThread:46677 [handler.py:handle_request():158] handle_request: partial_history
2024-05-28 11:51:12,892 DEBUG   HandlerThread:46677 [handler.py:handle_request():158] handle_request: partial_history
2024-05-28 11:51:13,265 DEBUG   HandlerThread:46677 [handler.py:handle_request():158] handle_request: partial_history
2024-05-28 11:51:13,649 DEBUG   HandlerThread:46677 [handler.py:handle_request():158] handle_request: partial_history
2024-05-28 11:51:14,103 DEBUG   HandlerThread:46677 [handler.py:handle_request():158] handle_request: partial_history
2024-05-28 11:51:14,554 DEBUG   HandlerThread:46677 [handler.py:handle_request():158] handle_request: partial_history
2024-05-28 11:51:15,023 DEBUG   HandlerThread:46677 [handler.py:handle_request():158] handle_request: partial_history
2024-05-28 11:51:15,563 DEBUG   HandlerThread:46677 [handler.py:handle_request():158] handle_request: partial_history
2024-05-28 11:51:16,076 DEBUG   HandlerThread:46677 [handler.py:handle_request():158] handle_request: partial_history
2024-05-28 11:51:16,435 DEBUG   HandlerThread:46677 [handler.py:handle_request():158] handle_request: partial_history
2024-05-28 11:51:16,645 DEBUG   HandlerThread:46677 [handler.py:handle_request():158] handle_request: keepalive
2024-05-28 11:51:16,792 DEBUG   HandlerThread:46677 [handler.py:handle_request():158] handle_request: partial_history
2024-05-28 11:51:16,949 DEBUG   HandlerThread:46677 [handler.py:handle_request():158] handle_request: partial_history
2024-05-28 11:51:17,304 DEBUG   HandlerThread:46677 [handler.py:handle_request():158] handle_request: partial_history
2024-05-28 11:51:17,674 DEBUG   HandlerThread:46677 [handler.py:handle_request():158] handle_request: partial_history
2024-05-28 11:51:18,175 DEBUG   HandlerThread:46677 [handler.py:handle_request():158] handle_request: partial_history
2024-05-28 11:51:18,673 DEBUG   HandlerThread:46677 [handler.py:handle_request():158] handle_request: partial_history
2024-05-28 11:51:19,201 DEBUG   HandlerThread:46677 [handler.py:handle_request():158] handle_request: partial_history
2024-05-28 11:51:19,603 DEBUG   HandlerThread:46677 [handler.py:handle_request():158] handle_request: partial_history
2024-05-28 11:51:20,092 DEBUG   SenderThread:46677 [sender.py:send():378] send: telemetry
2024-05-28 11:51:20,092 DEBUG   HandlerThread:46677 [handler.py:handle_request():158] handle_request: status_report
2024-05-28 11:51:20,104 DEBUG   SenderThread:46677 [sender.py:send():378] send: history
2024-05-28 11:51:20,104 DEBUG   SenderThread:46677 [sender.py:send_request():405] send_request: summary_record
2024-05-28 11:51:20,108 INFO    SenderThread:46677 [sender.py:_save_file():1389] saving file wandb-summary.json with policy end
2024-05-28 11:51:20,109 DEBUG   SenderThread:46677 [sender.py:send():378] send: history
2024-05-28 11:51:20,109 DEBUG   SenderThread:46677 [sender.py:send_request():405] send_request: summary_record
2024-05-28 11:51:20,110 INFO    SenderThread:46677 [sender.py:_save_file():1389] saving file wandb-summary.json with policy end
2024-05-28 11:51:20,110 DEBUG   SenderThread:46677 [sender.py:send():378] send: history
2024-05-28 11:51:20,110 DEBUG   SenderThread:46677 [sender.py:send_request():405] send_request: summary_record
2024-05-28 11:51:20,111 INFO    SenderThread:46677 [sender.py:_save_file():1389] saving file wandb-summary.json with policy end
2024-05-28 11:51:20,112 DEBUG   SenderThread:46677 [sender.py:send():378] send: history
2024-05-28 11:51:20,112 DEBUG   SenderThread:46677 [sender.py:send_request():405] send_request: summary_record
2024-05-28 11:51:20,113 INFO    SenderThread:46677 [sender.py:_save_file():1389] saving file wandb-summary.json with policy end
2024-05-28 11:51:20,113 DEBUG   SenderThread:46677 [sender.py:send():378] send: history
2024-05-28 11:51:20,113 DEBUG   SenderThread:46677 [sender.py:send_request():405] send_request: summary_record
2024-05-28 11:51:20,114 INFO    SenderThread:46677 [sender.py:_save_file():1389] saving file wandb-summary.json with policy end
2024-05-28 11:51:20,114 DEBUG   SenderThread:46677 [sender.py:send():378] send: history
2024-05-28 11:51:20,115 DEBUG   SenderThread:46677 [sender.py:send_request():405] send_request: summary_record
2024-05-28 11:51:20,117 INFO    SenderThread:46677 [sender.py:_save_file():1389] saving file wandb-summary.json with policy end
2024-05-28 11:51:20,117 DEBUG   SenderThread:46677 [sender.py:send():378] send: history
2024-05-28 11:51:20,118 DEBUG   SenderThread:46677 [sender.py:send_request():405] send_request: summary_record
2024-05-28 11:51:20,118 INFO    SenderThread:46677 [sender.py:_save_file():1389] saving file wandb-summary.json with policy end
2024-05-28 11:51:20,119 DEBUG   SenderThread:46677 [sender.py:send():378] send: history
2024-05-28 11:51:20,119 DEBUG   SenderThread:46677 [sender.py:send_request():405] send_request: summary_record
2024-05-28 11:51:20,120 INFO    SenderThread:46677 [sender.py:_save_file():1389] saving file wandb-summary.json with policy end
2024-05-28 11:51:20,121 DEBUG   SenderThread:46677 [sender.py:send():378] send: history
2024-05-28 11:51:20,121 DEBUG   SenderThread:46677 [sender.py:send_request():405] send_request: summary_record
2024-05-28 11:51:20,122 INFO    SenderThread:46677 [sender.py:_save_file():1389] saving file wandb-summary.json with policy end
2024-05-28 11:51:20,123 DEBUG   SenderThread:46677 [sender.py:send():378] send: history
2024-05-28 11:51:20,123 DEBUG   SenderThread:46677 [sender.py:send_request():405] send_request: summary_record
2024-05-28 11:51:20,124 INFO    SenderThread:46677 [sender.py:_save_file():1389] saving file wandb-summary.json with policy end
2024-05-28 11:51:20,124 DEBUG   SenderThread:46677 [sender.py:send():378] send: history
2024-05-28 11:51:20,125 DEBUG   SenderThread:46677 [sender.py:send_request():405] send_request: summary_record
2024-05-28 11:51:20,125 INFO    SenderThread:46677 [sender.py:_save_file():1389] saving file wandb-summary.json with policy end
2024-05-28 11:51:20,126 DEBUG   SenderThread:46677 [sender.py:send():378] send: history
2024-05-28 11:51:20,126 DEBUG   SenderThread:46677 [sender.py:send_request():405] send_request: summary_record
2024-05-28 11:51:20,127 DEBUG   HandlerThread:46677 [handler.py:handle_request():158] handle_request: partial_history
2024-05-28 11:51:20,129 INFO    SenderThread:46677 [sender.py:_save_file():1389] saving file wandb-summary.json with policy end
2024-05-28 11:51:20,129 DEBUG   SenderThread:46677 [sender.py:send():378] send: history
2024-05-28 11:51:20,129 DEBUG   SenderThread:46677 [sender.py:send_request():405] send_request: summary_record
2024-05-28 11:51:20,130 INFO    SenderThread:46677 [sender.py:_save_file():1389] saving file wandb-summary.json with policy end
2024-05-28 11:51:20,131 DEBUG   SenderThread:46677 [sender.py:send():378] send: history
2024-05-28 11:51:20,131 DEBUG   SenderThread:46677 [sender.py:send_request():405] send_request: summary_record
2024-05-28 11:51:20,132 INFO    SenderThread:46677 [sender.py:_save_file():1389] saving file wandb-summary.json with policy end
2024-05-28 11:51:20,132 DEBUG   SenderThread:46677 [sender.py:send():378] send: history
2024-05-28 11:51:20,134 DEBUG   SenderThread:46677 [sender.py:send_request():405] send_request: summary_record
2024-05-28 11:51:20,137 INFO    SenderThread:46677 [sender.py:_save_file():1389] saving file wandb-summary.json with policy end
2024-05-28 11:51:20,139 DEBUG   SenderThread:46677 [sender.py:send():378] send: history
2024-05-28 11:51:20,139 DEBUG   SenderThread:46677 [sender.py:send_request():405] send_request: summary_record
2024-05-28 11:51:20,140 INFO    SenderThread:46677 [sender.py:_save_file():1389] saving file wandb-summary.json with policy end
2024-05-28 11:51:20,141 DEBUG   SenderThread:46677 [sender.py:send():378] send: history
2024-05-28 11:51:20,141 DEBUG   SenderThread:46677 [sender.py:send_request():405] send_request: summary_record
2024-05-28 11:51:20,142 INFO    SenderThread:46677 [sender.py:_save_file():1389] saving file wandb-summary.json with policy end
2024-05-28 11:51:20,143 DEBUG   SenderThread:46677 [sender.py:send():378] send: history
2024-05-28 11:51:20,143 DEBUG   SenderThread:46677 [sender.py:send_request():405] send_request: summary_record
2024-05-28 11:51:20,144 INFO    SenderThread:46677 [sender.py:_save_file():1389] saving file wandb-summary.json with policy end
2024-05-28 11:51:20,144 DEBUG   SenderThread:46677 [sender.py:send():378] send: history
2024-05-28 11:51:20,144 DEBUG   SenderThread:46677 [sender.py:send_request():405] send_request: summary_record
2024-05-28 11:51:20,146 INFO    SenderThread:46677 [sender.py:_save_file():1389] saving file wandb-summary.json with policy end
2024-05-28 11:51:20,147 DEBUG   SenderThread:46677 [sender.py:send():378] send: history
2024-05-28 11:51:20,147 DEBUG   SenderThread:46677 [sender.py:send_request():405] send_request: summary_record
2024-05-28 11:51:20,147 INFO    SenderThread:46677 [sender.py:_save_file():1389] saving file wandb-summary.json with policy end
2024-05-28 11:51:20,148 DEBUG   SenderThread:46677 [sender.py:send():378] send: history
2024-05-28 11:51:20,148 DEBUG   SenderThread:46677 [sender.py:send_request():405] send_request: summary_record
2024-05-28 11:51:20,149 INFO    SenderThread:46677 [sender.py:_save_file():1389] saving file wandb-summary.json with policy end
2024-05-28 11:51:20,149 DEBUG   SenderThread:46677 [sender.py:send():378] send: history
2024-05-28 11:51:20,150 DEBUG   SenderThread:46677 [sender.py:send_request():405] send_request: summary_record
2024-05-28 11:51:20,151 INFO    SenderThread:46677 [sender.py:_save_file():1389] saving file wandb-summary.json with policy end
2024-05-28 11:51:20,152 DEBUG   SenderThread:46677 [sender.py:send():378] send: history
2024-05-28 11:51:20,152 DEBUG   SenderThread:46677 [sender.py:send_request():405] send_request: summary_record
2024-05-28 11:51:20,152 INFO    SenderThread:46677 [sender.py:_save_file():1389] saving file wandb-summary.json with policy end
2024-05-28 11:51:20,153 DEBUG   SenderThread:46677 [sender.py:send():378] send: history
2024-05-28 11:51:20,153 DEBUG   SenderThread:46677 [sender.py:send_request():405] send_request: summary_record
2024-05-28 11:51:20,154 INFO    SenderThread:46677 [sender.py:_save_file():1389] saving file wandb-summary.json with policy end
2024-05-28 11:51:20,155 DEBUG   SenderThread:46677 [sender.py:send():378] send: history
2024-05-28 11:51:20,155 DEBUG   SenderThread:46677 [sender.py:send_request():405] send_request: summary_record
2024-05-28 11:51:20,156 INFO    SenderThread:46677 [sender.py:_save_file():1389] saving file wandb-summary.json with policy end
2024-05-28 11:51:20,156 DEBUG   SenderThread:46677 [sender.py:send():378] send: history
2024-05-28 11:51:20,156 DEBUG   SenderThread:46677 [sender.py:send_request():405] send_request: summary_record
2024-05-28 11:51:20,157 INFO    SenderThread:46677 [sender.py:_save_file():1389] saving file wandb-summary.json with policy end
2024-05-28 11:51:20,158 DEBUG   SenderThread:46677 [sender.py:send():378] send: history
2024-05-28 11:51:20,158 DEBUG   SenderThread:46677 [sender.py:send_request():405] send_request: summary_record
2024-05-28 11:51:20,159 INFO    SenderThread:46677 [sender.py:_save_file():1389] saving file wandb-summary.json with policy end
2024-05-28 11:51:20,159 DEBUG   SenderThread:46677 [sender.py:send():378] send: history
2024-05-28 11:51:20,159 DEBUG   SenderThread:46677 [sender.py:send_request():405] send_request: summary_record
2024-05-28 11:51:20,160 INFO    SenderThread:46677 [sender.py:_save_file():1389] saving file wandb-summary.json with policy end
2024-05-28 11:51:20,161 DEBUG   SenderThread:46677 [sender.py:send():378] send: history
2024-05-28 11:51:20,161 DEBUG   SenderThread:46677 [sender.py:send_request():405] send_request: summary_record
2024-05-28 11:51:20,161 INFO    SenderThread:46677 [sender.py:_save_file():1389] saving file wandb-summary.json with policy end
2024-05-28 11:51:20,162 DEBUG   SenderThread:46677 [sender.py:send():378] send: history
2024-05-28 11:51:20,162 DEBUG   SenderThread:46677 [sender.py:send_request():405] send_request: summary_record
2024-05-28 11:51:20,163 INFO    SenderThread:46677 [sender.py:_save_file():1389] saving file wandb-summary.json with policy end
2024-05-28 11:51:20,164 DEBUG   SenderThread:46677 [sender.py:send():378] send: history
2024-05-28 11:51:20,164 DEBUG   SenderThread:46677 [sender.py:send_request():405] send_request: summary_record
2024-05-28 11:51:20,165 INFO    SenderThread:46677 [sender.py:_save_file():1389] saving file wandb-summary.json with policy end
2024-05-28 11:51:20,619 INFO    Thread-12 :46677 [dir_watcher.py:_on_file_created():271] file/dir created: /home/alessiatortone/ProjectMLDL/wandb/run-20240528_115104-kybjttf5/files/wandb-summary.json
2024-05-28 11:51:20,620 INFO    Thread-12 :46677 [dir_watcher.py:_on_file_created():271] file/dir created: /home/alessiatortone/ProjectMLDL/wandb/run-20240528_115104-kybjttf5/files/output.log
2024-05-28 11:51:20,762 DEBUG   HandlerThread:46677 [handler.py:handle_request():158] handle_request: partial_history
2024-05-28 11:51:20,764 DEBUG   SenderThread:46677 [sender.py:send():378] send: history
2024-05-28 11:51:20,764 DEBUG   SenderThread:46677 [sender.py:send_request():405] send_request: summary_record
2024-05-28 11:51:20,765 INFO    SenderThread:46677 [sender.py:_save_file():1389] saving file wandb-summary.json with policy end
2024-05-28 11:51:21,274 DEBUG   HandlerThread:46677 [handler.py:handle_request():158] handle_request: partial_history
2024-05-28 11:51:21,276 DEBUG   SenderThread:46677 [sender.py:send():378] send: history
2024-05-28 11:51:21,276 DEBUG   SenderThread:46677 [sender.py:send_request():405] send_request: summary_record
2024-05-28 11:51:21,277 INFO    SenderThread:46677 [sender.py:_save_file():1389] saving file wandb-summary.json with policy end
2024-05-28 11:51:21,609 DEBUG   HandlerThread:46677 [handler.py:handle_request():158] handle_request: stop_status
2024-05-28 11:51:21,609 DEBUG   SenderThread:46677 [sender.py:send_request():405] send_request: stop_status
2024-05-28 11:51:21,621 INFO    Thread-12 :46677 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/alessiatortone/ProjectMLDL/wandb/run-20240528_115104-kybjttf5/files/wandb-summary.json
2024-05-28 11:51:21,651 DEBUG   HandlerThread:46677 [handler.py:handle_request():158] handle_request: partial_history
2024-05-28 11:51:22,092 DEBUG   HandlerThread:46677 [handler.py:handle_request():158] handle_request: partial_history
2024-05-28 11:51:22,591 DEBUG   HandlerThread:46677 [handler.py:handle_request():158] handle_request: partial_history
2024-05-28 11:51:22,622 INFO    Thread-12 :46677 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/alessiatortone/ProjectMLDL/wandb/run-20240528_115104-kybjttf5/files/output.log
2024-05-28 11:51:23,092 DEBUG   HandlerThread:46677 [handler.py:handle_request():158] handle_request: partial_history
2024-05-28 11:51:23,579 DEBUG   HandlerThread:46677 [handler.py:handle_request():158] handle_request: partial_history
2024-05-28 11:51:24,049 DEBUG   HandlerThread:46677 [handler.py:handle_request():158] handle_request: partial_history
2024-05-28 11:51:24,680 DEBUG   HandlerThread:46677 [handler.py:handle_request():158] handle_request: partial_history
2024-05-28 11:51:25,192 DEBUG   HandlerThread:46677 [handler.py:handle_request():158] handle_request: partial_history
2024-05-28 11:51:25,714 DEBUG   HandlerThread:46677 [handler.py:handle_request():158] handle_request: partial_history
2024-05-28 11:51:26,261 DEBUG   HandlerThread:46677 [handler.py:handle_request():158] handle_request: partial_history
2024-05-28 11:51:26,689 DEBUG   HandlerThread:46677 [handler.py:handle_request():158] handle_request: keepalive
2024-05-28 11:51:26,711 DEBUG   HandlerThread:46677 [handler.py:handle_request():158] handle_request: partial_history
2024-05-28 11:51:27,074 DEBUG   HandlerThread:46677 [handler.py:handle_request():158] handle_request: partial_history
2024-05-28 11:51:27,456 DEBUG   HandlerThread:46677 [handler.py:handle_request():158] handle_request: partial_history
2024-05-28 11:51:27,944 DEBUG   HandlerThread:46677 [handler.py:handle_request():158] handle_request: partial_history
2024-05-28 11:51:28,625 DEBUG   HandlerThread:46677 [handler.py:handle_request():158] handle_request: partial_history
2024-05-28 11:51:29,124 DEBUG   HandlerThread:46677 [handler.py:handle_request():158] handle_request: partial_history
2024-05-28 11:51:29,636 DEBUG   HandlerThread:46677 [handler.py:handle_request():158] handle_request: partial_history
2024-05-28 11:51:30,640 DEBUG   HandlerThread:46677 [handler.py:handle_request():158] handle_request: poll_exit
2024-05-28 11:51:35,643 DEBUG   HandlerThread:46677 [handler.py:handle_request():158] handle_request: keepalive
2024-05-28 11:51:40,647 DEBUG   HandlerThread:46677 [handler.py:handle_request():158] handle_request: keepalive
2024-05-28 11:51:45,649 DEBUG   HandlerThread:46677 [handler.py:handle_request():158] handle_request: keepalive
2024-05-28 11:51:50,652 DEBUG   HandlerThread:46677 [handler.py:handle_request():158] handle_request: keepalive
2024-05-28 11:51:55,655 DEBUG   HandlerThread:46677 [handler.py:handle_request():158] handle_request: keepalive
2024-05-28 11:52:00,716 DEBUG   HandlerThread:46677 [handler.py:handle_request():158] handle_request: keepalive
2024-05-28 11:52:04,733 DEBUG   SystemMonitor:46677 [system_monitor.py:_start():172] Starting system metrics aggregation loop
2024-05-28 11:52:05,715 DEBUG   HandlerThread:46677 [handler.py:handle_request():158] handle_request: keepalive
2024-05-28 11:52:10,719 DEBUG   HandlerThread:46677 [handler.py:handle_request():158] handle_request: keepalive
2024-05-28 11:52:15,723 DEBUG   HandlerThread:46677 [handler.py:handle_request():158] handle_request: keepalive
2024-05-28 11:52:20,727 DEBUG   HandlerThread:46677 [handler.py:handle_request():158] handle_request: keepalive
2024-05-28 11:52:25,295 INFO    SenderThread:46677 [retry.py:__call__():172] Retry attempt failed:
Traceback (most recent call last):
  File "/home/alessiatortone/miniconda3/envs/mldl/lib/python3.7/site-packages/urllib3/connection.py", line 207, in _new_conn
    socket_options=self.socket_options,
  File "/home/alessiatortone/miniconda3/envs/mldl/lib/python3.7/site-packages/urllib3/util/connection.py", line 60, in create_connection
    for res in socket.getaddrinfo(host, port, family, socket.SOCK_STREAM):
  File "/home/alessiatortone/miniconda3/envs/mldl/lib/python3.7/socket.py", line 752, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
socket.gaierror: [Errno -3] Temporary failure in name resolution

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/alessiatortone/miniconda3/envs/mldl/lib/python3.7/site-packages/urllib3/connectionpool.py", line 803, in urlopen
    **response_kw,
  File "/home/alessiatortone/miniconda3/envs/mldl/lib/python3.7/site-packages/urllib3/connectionpool.py", line 492, in _make_request
    raise new_e
  File "/home/alessiatortone/miniconda3/envs/mldl/lib/python3.7/site-packages/urllib3/connectionpool.py", line 468, in _make_request
    self._validate_conn(conn)
  File "/home/alessiatortone/miniconda3/envs/mldl/lib/python3.7/site-packages/urllib3/connectionpool.py", line 1097, in _validate_conn
    conn.connect()
  File "/home/alessiatortone/miniconda3/envs/mldl/lib/python3.7/site-packages/urllib3/connection.py", line 611, in connect
    self.sock = sock = self._new_conn()
  File "/home/alessiatortone/miniconda3/envs/mldl/lib/python3.7/site-packages/urllib3/connection.py", line 210, in _new_conn
    raise NameResolutionError(self.host, self, e) from e
urllib3.exceptions.NameResolutionError: <urllib3.connection.HTTPSConnection object at 0x7f60e1707e50>: Failed to resolve 'api.wandb.ai' ([Errno -3] Temporary failure in name resolution)

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/alessiatortone/miniconda3/envs/mldl/lib/python3.7/site-packages/requests/adapters.py", line 497, in send
    chunked=chunked,
  File "/home/alessiatortone/miniconda3/envs/mldl/lib/python3.7/site-packages/urllib3/connectionpool.py", line 846, in urlopen
    method, url, error=new_e, _pool=self, _stacktrace=sys.exc_info()[2]
  File "/home/alessiatortone/miniconda3/envs/mldl/lib/python3.7/site-packages/urllib3/util/retry.py", line 515, in increment
    raise MaxRetryError(_pool, url, reason) from reason  # type: ignore[arg-type]
urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /graphql (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f60e1707e50>: Failed to resolve 'api.wandb.ai' ([Errno -3] Temporary failure in name resolution)"))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/alessiatortone/miniconda3/envs/mldl/lib/python3.7/site-packages/wandb/sdk/lib/retry.py", line 131, in __call__
    result = self._call_fn(*args, **kwargs)
  File "/home/alessiatortone/miniconda3/envs/mldl/lib/python3.7/site-packages/wandb/sdk/internal/internal_api.py", line 340, in execute
    return self.client.execute(*args, **kwargs)  # type: ignore
  File "/home/alessiatortone/miniconda3/envs/mldl/lib/python3.7/site-packages/wandb/vendor/gql-0.2.0/wandb_gql/client.py", line 52, in execute
    result = self._get_result(document, *args, **kwargs)
  File "/home/alessiatortone/miniconda3/envs/mldl/lib/python3.7/site-packages/wandb/vendor/gql-0.2.0/wandb_gql/client.py", line 60, in _get_result
    return self.transport.execute(document, *args, **kwargs)
  File "/home/alessiatortone/miniconda3/envs/mldl/lib/python3.7/site-packages/wandb/sdk/lib/gql_request.py", line 58, in execute
    request = self.session.post(self.url, **post_args)
  File "/home/alessiatortone/miniconda3/envs/mldl/lib/python3.7/site-packages/requests/sessions.py", line 637, in post
    return self.request("POST", url, data=data, json=json, **kwargs)
  File "/home/alessiatortone/miniconda3/envs/mldl/lib/python3.7/site-packages/requests/sessions.py", line 589, in request
    resp = self.send(prep, **send_kwargs)
  File "/home/alessiatortone/miniconda3/envs/mldl/lib/python3.7/site-packages/requests/sessions.py", line 703, in send
    r = adapter.send(request, **kwargs)
  File "/home/alessiatortone/miniconda3/envs/mldl/lib/python3.7/site-packages/requests/adapters.py", line 519, in send
    raise ConnectionError(e, request=request)
requests.exceptions.ConnectionError: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /graphql (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f60e1707e50>: Failed to resolve 'api.wandb.ai' ([Errno -3] Temporary failure in name resolution)"))
2024-05-28 11:52:25,728 DEBUG   HandlerThread:46677 [handler.py:handle_request():158] handle_request: keepalive
2024-05-28 11:52:30,733 DEBUG   HandlerThread:46677 [handler.py:handle_request():158] handle_request: keepalive
2024-05-28 11:52:35,735 DEBUG   HandlerThread:46677 [handler.py:handle_request():158] handle_request: keepalive
2024-05-28 11:52:40,739 DEBUG   HandlerThread:46677 [handler.py:handle_request():158] handle_request: keepalive
2024-05-28 11:52:45,741 DEBUG   HandlerThread:46677 [handler.py:handle_request():158] handle_request: keepalive
2024-05-28 11:52:50,744 DEBUG   HandlerThread:46677 [handler.py:handle_request():158] handle_request: keepalive
2024-05-28 11:52:55,748 DEBUG   HandlerThread:46677 [handler.py:handle_request():158] handle_request: keepalive
2024-05-28 11:53:00,751 DEBUG   HandlerThread:46677 [handler.py:handle_request():158] handle_request: keepalive
2024-05-28 11:53:05,754 DEBUG   HandlerThread:46677 [handler.py:handle_request():158] handle_request: keepalive
2024-05-28 11:53:10,757 DEBUG   HandlerThread:46677 [handler.py:handle_request():158] handle_request: keepalive
2024-05-28 11:53:15,760 DEBUG   HandlerThread:46677 [handler.py:handle_request():158] handle_request: keepalive
2024-05-28 11:53:20,762 DEBUG   HandlerThread:46677 [handler.py:handle_request():158] handle_request: keepalive
2024-05-28 11:53:25,765 DEBUG   HandlerThread:46677 [handler.py:handle_request():158] handle_request: keepalive
2024-05-28 11:53:30,769 DEBUG   HandlerThread:46677 [handler.py:handle_request():158] handle_request: keepalive
2024-05-28 11:53:35,772 DEBUG   HandlerThread:46677 [handler.py:handle_request():158] handle_request: keepalive
2024-05-28 11:53:40,773 DEBUG   HandlerThread:46677 [handler.py:handle_request():158] handle_request: keepalive
2024-05-28 11:53:45,273 WARNING FileStreamThread:46677 [file_stream.py:request_with_retry():679] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/mldl_alessia/calGTT/kybjttf5/file_stream (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f60e13cef90>: Failed to establish a new connection: [Errno 110] Connection timed out')). func: functools.partial(<bound method Session.post of <requests.sessions.Session object at 0x7f60e1f699d0>>, timeout=180.0), args: ('https://api.wandb.ai/files/mldl_alessia/calGTT/kybjttf5/file_stream',), kwargs: {'json': {'files': {'wandb-summary.json': {'offset': 0, 'content': ['{"Reward": 369.4884759633961, "_timestamp": 1716889867.6947517, "_runtime": 3.639058828353882, "_step": 0}']}, 'wandb-history.jsonl': {'offset': 0, 'content': ['{"Reward": 369.4884759633961, "_timestamp": 1716889867.6947517, "_runtime": 3.639058828353882, "_step": 0}']}}, 'dropped': 0}}
2024-05-28 11:53:45,776 DEBUG   HandlerThread:46677 [handler.py:handle_request():158] handle_request: keepalive
2024-05-28 11:53:50,781 DEBUG   HandlerThread:46677 [handler.py:handle_request():158] handle_request: keepalive
2024-05-28 11:53:52,865 INFO    memory    :46677 [interfaces.py:monitor():138] Process proc.memory.rssMB has exited.
2024-05-28 11:53:52,866 DEBUG   SystemMonitor:46677 [system_monitor.py:_start():179] Finished system metrics aggregation loop
2024-05-28 11:53:52,866 DEBUG   SystemMonitor:46677 [system_monitor.py:_start():183] Publishing last batch of metrics
2024-05-28 11:53:53,282 WARNING StreamThr :46677 [internal.py:is_dead():414] Internal process exiting, parent pid 46634 disappeared
2024-05-28 11:53:53,282 ERROR   StreamThr :46677 [internal.py:wandb_internal():151] Internal process shutdown.
2024-05-28 11:53:53,869 INFO    HandlerThread:46677 [handler.py:finish():882] shutting down handler
2024-05-28 11:53:53,870 INFO    HandlerThread:46677 [system_monitor.py:finish():203] Stopping system monitor
2024-05-28 11:53:53,870 INFO    HandlerThread:46677 [interfaces.py:finish():200] Joined cpu monitor
2024-05-28 11:53:53,871 INFO    HandlerThread:46677 [interfaces.py:finish():200] Joined disk monitor
2024-05-28 11:53:53,872 INFO    HandlerThread:46677 [interfaces.py:finish():200] Joined memory monitor
2024-05-28 11:53:53,873 INFO    HandlerThread:46677 [interfaces.py:finish():200] Joined network monitor
2024-05-28 11:53:53,874 INFO    WriterThread:46677 [datastore.py:close():296] close: /home/alessiatortone/ProjectMLDL/wandb/run-20240528_115104-kybjttf5/run-kybjttf5.wandb
2024-05-28 11:53:57,583 WARNING FileStreamThread:46677 [file_stream.py:request_with_retry():679] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/mldl_alessia/calGTT/kybjttf5/file_stream (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f60e12ce310>: Failed to resolve 'api.wandb.ai' ([Errno -3] Temporary failure in name resolution)")). func: functools.partial(<bound method Session.post of <requests.sessions.Session object at 0x7f60e1f699d0>>, timeout=180.0), args: ('https://api.wandb.ai/files/mldl_alessia/calGTT/kybjttf5/file_stream',), kwargs: {'json': {'files': {'wandb-summary.json': {'offset': 0, 'content': ['{"Reward": 369.4884759633961, "_timestamp": 1716889867.6947517, "_runtime": 3.639058828353882, "_step": 0}']}, 'wandb-history.jsonl': {'offset': 0, 'content': ['{"Reward": 369.4884759633961, "_timestamp": 1716889867.6947517, "_runtime": 3.639058828353882, "_step": 0}']}}, 'dropped': 0}}
2024-05-28 11:54:11,761 WARNING FileStreamThread:46677 [file_stream.py:request_with_retry():679] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/mldl_alessia/calGTT/kybjttf5/file_stream (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f60e12d0850>: Failed to resolve 'api.wandb.ai' ([Errno -3] Temporary failure in name resolution)")). func: functools.partial(<bound method Session.post of <requests.sessions.Session object at 0x7f60e1f699d0>>, timeout=180.0), args: ('https://api.wandb.ai/files/mldl_alessia/calGTT/kybjttf5/file_stream',), kwargs: {'json': {'files': {'wandb-summary.json': {'offset': 0, 'content': ['{"Reward": 369.4884759633961, "_timestamp": 1716889867.6947517, "_runtime": 3.639058828353882, "_step": 0}']}, 'wandb-history.jsonl': {'offset': 0, 'content': ['{"Reward": 369.4884759633961, "_timestamp": 1716889867.6947517, "_runtime": 3.639058828353882, "_step": 0}']}}, 'dropped': 0}}
2024-05-28 11:54:30,027 WARNING FileStreamThread:46677 [file_stream.py:request_with_retry():679] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/mldl_alessia/calGTT/kybjttf5/file_stream (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f60e170cd50>: Failed to resolve 'api.wandb.ai' ([Errno -3] Temporary failure in name resolution)")). func: functools.partial(<bound method Session.post of <requests.sessions.Session object at 0x7f60e1f699d0>>, timeout=180.0), args: ('https://api.wandb.ai/files/mldl_alessia/calGTT/kybjttf5/file_stream',), kwargs: {'json': {'files': {'wandb-summary.json': {'offset': 0, 'content': ['{"Reward": 369.4884759633961, "_timestamp": 1716889867.6947517, "_runtime": 3.639058828353882, "_step": 0}']}, 'wandb-history.jsonl': {'offset': 0, 'content': ['{"Reward": 369.4884759633961, "_timestamp": 1716889867.6947517, "_runtime": 3.639058828353882, "_step": 0}']}}, 'dropped': 0}}
2024-05-28 11:54:58,663 WARNING FileStreamThread:46677 [file_stream.py:request_with_retry():679] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/mldl_alessia/calGTT/kybjttf5/file_stream (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f60e12cb510>: Failed to resolve 'api.wandb.ai' ([Errno -3] Temporary failure in name resolution)")). func: functools.partial(<bound method Session.post of <requests.sessions.Session object at 0x7f60e1f699d0>>, timeout=180.0), args: ('https://api.wandb.ai/files/mldl_alessia/calGTT/kybjttf5/file_stream',), kwargs: {'json': {'files': {'wandb-summary.json': {'offset': 0, 'content': ['{"Reward": 369.4884759633961, "_timestamp": 1716889867.6947517, "_runtime": 3.639058828353882, "_step": 0}']}, 'wandb-history.jsonl': {'offset': 0, 'content': ['{"Reward": 369.4884759633961, "_timestamp": 1716889867.6947517, "_runtime": 3.639058828353882, "_step": 0}']}}, 'dropped': 0}}
2024-05-28 11:55:44,046 WARNING FileStreamThread:46677 [file_stream.py:request_with_retry():679] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/mldl_alessia/calGTT/kybjttf5/file_stream (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f60e12c96d0>: Failed to resolve 'api.wandb.ai' ([Errno -3] Temporary failure in name resolution)")). func: functools.partial(<bound method Session.post of <requests.sessions.Session object at 0x7f60e1f699d0>>, timeout=180.0), args: ('https://api.wandb.ai/files/mldl_alessia/calGTT/kybjttf5/file_stream',), kwargs: {'json': {'files': {'wandb-summary.json': {'offset': 0, 'content': ['{"Reward": 369.4884759633961, "_timestamp": 1716889867.6947517, "_runtime": 3.639058828353882, "_step": 0}']}, 'wandb-history.jsonl': {'offset': 0, 'content': ['{"Reward": 369.4884759633961, "_timestamp": 1716889867.6947517, "_runtime": 3.639058828353882, "_step": 0}']}}, 'dropped': 0}}
2024-05-28 11:55:50,515 INFO    wandb-upload_0:46677 [upload_job.py:push():130] Uploaded file /tmp/tmp6mi99a07wandb/rqlwn8e5-wandb-metadata.json
2024-05-28 11:55:50,516 INFO    MainThread:46677 [internal.py:handle_exit():75] Internal process exited
